src/sentry/web/frontend/react_page.py

```diff
@@ -122,10 +120,11 @@ def handle_auth_required(self, request: Request, *args, **kwargs) -> HttpRespons
 
     def handle(self, request: Request, organization, **kwargs) -> HttpResponse:
         if "project_id" in kwargs and request.GET.get("onboarding"):
-            project = Project.objects.filter(
-                organization=organization, slug=kwargs["project_id"]
-            ).first()
-            first_event_pending.send(project=project, user=request.user, sender=self)
```

Comment:
We should also be able to fully delete this event + receiver now, as they're really not used anywhere else as far as I could tell.